Main.sublime-menu

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "id": "preferences",
+        "children": [
+            {
+                "id": "package-settings",
+                "children": [
+                    {
+                        // "caption": "Rainbowth",
+                        // "children": [
+                        //     {
+                        //         "caption": "Settings",
+                        //         "command": "edit_settings",
+                        //         "args": {
+                        //             "base_file": "${packages}/Rainbowth/Rainbowth.sublime-settings",
+                        //             "default": "// Any settings in this file overrides the default ones\n{\n\t$0\n}\n"
+                        //         }
+                        //     }
+                        // ]
```

Comment:
Stray comment?

Reply:
This is the code for putting it in a submenu. Currently it is like

```
Sublime Text >
    Preferences >
        Package Settings >
            Random Package 1
            Random Package 2 >
                Readme 
                Settings
            Rainbowth
```

But if you want to change the menu to include a readme or other options you’d have it like

```
Sublime Text >
    Preferences >
        Package Settings >
            Random Package 1
            Random Package 2 >
                Readme 
                Settings
            Rainbowth  >
                Readme 
                Settings
```

Which is the code that is in commented. I thought I’d leave it in so it is quicker and easier to amend this in the future. But you or I can get rid of it, if you want.